src/main/resources/application.properties

```diff
@@ -1,10 +1,10 @@
 #for local db (swagger,postman etc)
 
-spring.datasource.url=jdbc:h2:mem:demo;DB_CLOSE_ON_EXIT=FALSE
-spring.datasource.username=sa
-spring.datasource.password=
-spring.datasource.driver-class-name=org.h2.Driver
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+#spring.datasource.url=jdbc:h2:mem:demo;DB_CLOSE_ON_EXIT=FALSE
```

Comment:
Please, revert ANY changes in application.properties

Reply:
i dont know how its possible, but i dont have this red part of code in my prop file

Reply:
i deleted red part of code and lieve only green line. but i cheked and i see, master has this part of code